live-site/lib/types.ts

```diff
@@ -32,12 +32,32 @@ export interface EventScheduleTabProps {
 export interface StyledTabTitleWrapperProps {
   isSelected: boolean;
 }
+export interface EventItemTagProps{
+  type:string
+}
+
+export interface StyledTagAndTagTextProps{
+  type:string
+}
+
+export interface StyledArrowProps{
+  isOpen?: boolean
+}
 
 export interface EventItem {
+  id: number
   time: string;
   eventType: string;
   eventName: string;
   eventLocation: string;
+  description: string;
+  meal?: boolean;
+  beginner?: boolean;
+  workshop? : boolean;
+  actionItem?:boolean;
+  everyone?: boolean;
+  sponsor?: boolean;
+  social?: boolean;
```

Comment:
hmm maybe its possible to have one field here instead of many?
we could define an enum first:
```typescript
enum Tag {
  Meal = 'Meal',
  ActionItem = 'Action Item',
  ...
}
```
and then we can replace all these fields in eventItem with `tags: Tag[]`
it'll also make the code in `EventItem.tsx` cleaner, as you can loop through each possible tag instead of checking each one by one:
```typescript
{ eventItem.tags.map( (tag) => (<EventItemTag type={tag} />))}
```
this makes it much easier to add new tags, as we would only have to change the enum.